config/jobs/update-github-teams/peribolos-syncer-libs.yaml

```diff
@@ -0,0 +1,57 @@
+postsubmits:
+  falcosecurity/libs:
+  - name: peribolos-syncer-post-submit
+    branches:
+    - ^main$
+    decorate: true
+    max_concurrency: 1
+    skip_report: false
+    run_if_changed: 'OWNERS$'
+    spec:
+      containers:
+      - image: ghcr.io/falcosecurity/peribolos-syncer:0.2.0
+        args:
+        - sync
+        - github
+        - --org=falcosecurity
+        - --team=libs-maintainers
+        - --peribolos-config-path=config/org.yaml
+        - --peribolos-config-repository=test-infra
+        - --peribolos-config-git-ref=master
+        - --owners-repository=libs
+        - --owners-git-ref=main
```

Comment:
Shouldn't this be `master` for eg:  driverkit, libs, falco... repositories?

Reply:
You're right, good catch! Thanks

Reply:
@FedeDP it should be aligned now.

Plus, I've removed configurations for the libsinsp and the libscap repos.